.github/.dockstore.yml

```
@@ -161,3 +161,12 @@ workflows:
         - main
       tags:
         - /.*/
+
+  - subclass: WDL
+    name: SingleSample
```

```suggestion
    name: SingleSamplePipeline
```
How about this? Our workflow names usually contain a verb

---

Agree, that reads even better!